js/views/components/CryptoCurSelector.js

```diff
@@ -1,35 +1,11 @@
-import _ from 'underscore';
 import $ from 'jquery';
 import app from '../../app';
 import loadTemplate from '../../utils/loadTemplate';
-import { polyTFallback } from '../../utils/templateHelpers';
 import { isSupportedWalletCur } from '../../data/walletCurrencies';
 import baseVw from '../baseVw';
 
 export default class extends baseVw {
   constructor(options = {}) {
-    const controlTypes = ['checkbox', 'radio'];
```

Comment:
I moved these state data integrity checks to set state, so basically the rest of set state could assume good data instead of always checking.